blinky-config/src/test/java/org/spideruci/analysis/config/definer/TestConfigFieldsDefiner.java

```diff
@@ -130,7 +130,7 @@ public void testConfigClassScanner() {
       
       assertEquals(isFieldExpectedToBeDefined, isFieldActuallyDefined);
       
-      if(isFieldExpectedToBeDefined ^ isFieldExpectedToBeDefined) { // different
+      if(isFieldExpectedToBeDefined ^ isFieldActuallyDefined) { // different
```

Comment:
Can you include the printf logs that show in the terminal, when this passes? Also, consider changing the def/undef values in `publicStaticFieldNames` to make the test fail, and include those logs as well?

Reply:
Below the full code after I change `{ "inclusionList", "def" }` to `{ "inclusionList", "undef" }`. As you can see I think the print statements are useful. (This is after I move assertEquals(isFieldExpectedToBeDefined, isFieldActuallyDefined); below the if-else statement). I directly shows where the errors are coming from.

```
Running org.spideruci.analysis.config.definer.TestConfigFieldsDefiner
[debug] Public, static field UNDEFINED present in org/spideruci/analysis/statik/instrumentation/Deputy
[debug] Public, static field UNDEFINED not yet defined in org/spideruci/analysis/statik/instrumentation/Deputy
FIELD_NAME, Presence Check, (un)Defined Check
exclusionList Present
inclusionList Present expecting not defined; actually defined
[debug] Public, static field UNDEFINED present in org/spideruci/analysis/statik/instrumentation/Deputy
[debug] Public, static field UNDEFINED not yet defined in org/spideruci/analysis/statik/instrumentation/Deputy
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.006 sec <<< FAILURE!
testConfigClassScanner(org.spideruci.analysis.config.definer.TestConfigFieldsDefiner)  Time elapsed: 0.004 sec  <<< FAILURE!
java.lang.AssertionError: expected:<false> but was:<true>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at org.spideruci.analysis.config.definer.TestConfigFieldsDefiner.testConfigClassScanner(TestConfigFieldsDefiner.java:138)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
        at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
        at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```